README.md

```diff
@@ -83,26 +85,34 @@ Usage of ./endlessh-go
         comma-separated list of pattern=N settings for file-filtered logging
 ```
 
+## Using privileged ports (<1024)
```

Comment:
This is a docker problem not the endlessh-go problem.
The only place I mention docker in this main README.md is line 28.

This could go to the docker example.

You cannot assume every one is using container.
You cannot assume who is using container also uses docker.